llvm/lib/CodeGen/TargetLoweringObjectFileImpl.cpp

```diff
@@ -2767,15 +2774,55 @@ MCSection *TargetLoweringObjectFileGOFF::getExplicitSectionGlobal(
 MCSection *TargetLoweringObjectFileGOFF::getSectionForLSDA(
     const Function &F, const MCSymbol &FnSym, const TargetMachine &TM) const {
   std::string Name = ".gcc_exception_table." + F.getName().str();
-  return getContext().getGOFFSection(Name, SectionKind::getData(), nullptr, 0);
+  constexpr bool Is64Bit = true;
+  constexpr bool UsesXPLINK = true;
```

Comment:
These are of course always true in the current upstream code base.  I'm not sure it makes sense to add support for features not (currently) usable upstream ...

Reply:
My goal was to mark those values somehow, because it is very difficult to consistently find those spots again. But using a constant is enough, so this is gone now.